tests/test_settings.py

```diff
@@ -390,6 +391,10 @@ def test_ui_url_set_directly(self):
         with temporary_settings({PREFECT_UI_URL: "test"}):
             assert PREFECT_UI_URL.value() == "test"
 
+    def test_keep_alive_timeout_on_set(self):
+        with temporary_settings({PREFECT_ORION_API_KEEPALIVE: 30}):
+            assert PREFECT_ORION_API_KEEPALIVE.value() == 30
+
```

Comment:
I appreciate that you added coverage, but we don't need this test. These tests are only to cover settings with more complex behavior and this one is just a raw value and should be covered by tests of our `Setting` abstraction.

Reply:
I have removed this test